mdoc/Mono.Documentation/Updater/AttributeParserContext.cs

```diff
@@ -82,6 +85,11 @@ public string GetTupleElementName()
             return (tupleElementNames == null || tupleNameAttributeIndex >= tupleElementNames.Length) ? null : tupleElementNames[tupleNameAttributeIndex++];
         }
 
+        public bool IsNativeInteger()
+        {
+            return (nativeIntegerFlags == null || nativeIntegerAttributeIndex >= nativeIntegerFlags.Length) ? false : nativeIntegerFlags[nativeIntegerAttributeIndex++];
```

Comment:
Is "return nativeIntegerFlags != null && nativeIntegerAttributeIndex < nativeIntegerFlags.Length) && nativeIntegerFlags[nativeIntegerAttributeIndex++]; " better?

Reply:
Updated